packages/burner-wallet-connector/src/tests/burnerWalletConnector.test.ts

```diff
@@ -0,0 +1,231 @@
+import path from 'node:path';
+import { launchNodeAndGetWallets } from '@fuel-ts/account/test-utils';
+import type { Asset, Network, Provider } from 'fuels';
+import {
+  afterAll,
+  beforeAll,
+  beforeEach,
+  describe,
+  expect,
+  test,
+} from 'vitest';
+import { BurnerWalletConnector } from '../BurnerWalletConnector';
+
+describe('Burner Wallet Connector', () => {
+  let fuelProvider: Provider;
+  let connector: BurnerWalletConnector;
+
+  let stopProvider: () => void;
+
+  const chainConfigPath = path.join(__dirname, 'chainConfig.json');
+
+  beforeAll(async () => {
+    process.env.GENESIS_SECRET =
+      '0x6e48a022f9d4ae187bca4e2645abd62198ae294ee484766edbdaadf78160dc68';
+    const { stop, provider } = await launchNodeAndGetWallets({
+      launchNodeOptions: {
+        args: ['--chain', chainConfigPath],
+        loggingEnabled: false,
+      },
+    });
+
+    fuelProvider = provider;
+    stopProvider = stop;
+  });
+
+  afterAll(() => {
+    stopProvider?.();
+  });
+
+  beforeEach(async () => {
+    connector = new BurnerWalletConnector({
+      fuelProvider,
+    });
```

Comment:
We should have a test passing a custom storage to test if the feature is working.